pkg/app/master/command/images/model.go

```diff
@@ -0,0 +1,166 @@
+package images
+
+import (
+	"time"
+
+	"github.com/charmbracelet/bubbles/key"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/charmbracelet/lipgloss/table"
+	"github.com/dustin/go-humanize"
+	"github.com/mintoolkit/mint/pkg/app"
+
+	"github.com/mintoolkit/mint/pkg/app/master/command"
+	"github.com/mintoolkit/mint/pkg/app/master/tui/common"
+	"github.com/mintoolkit/mint/pkg/app/master/tui/keys"
+	"github.com/mintoolkit/mint/pkg/crt"
+	"github.com/mintoolkit/mint/pkg/crt/docker/dockerutil"
+
+	tea "github.com/charmbracelet/bubbletea"
+)
+
+// Model represents the state of the TUI.
+type Model struct {
```

Comment:
Here as well... let's use a less Bubble Tea specific name... Let's call it `TUI` (`type TUI struct`). This way, somebody who doesn't know much about the Bubble Tea architecture and terminology will be able to understand what this code is for. Comments help too, of course. Calling the struct `TUI` will make it more obvious.

Reply:
Noted and addressed.